src/CocktailDetails/CocktailDetails.tsx

```diff
@@ -95,40 +96,16 @@ const CocktailDetails: React.FC<CocktailDetailsProps> = ({ id, favCocktails, mad
           </section>
 					<section className='button-container'>
 						{!isFavorite && (
-							<button
-								type="button"
-								aria-label="add-to-favorites"
-								onClick={() => clickHandler(setIsFavorite, isFavorite, favCocktails, setFavCocktails)}
-							>
-								Add to Favorites
-							</button>
+                <img src={require("../assets/heart.svg")} alt="Add to Favorites" className="icon" onClick={() => clickHandler(setIsFavorite, isFavorite, favCocktails, setFavCocktails)}/>
```

Comment:
you have to require local images when using react and webpack

Reply:
So good to know! Thanks for figuring this out for us!

Reply:
good find alex